android/app/build.gradle

```diff
@@ -208,14 +208,14 @@ android {
         dev {
             minSdkVersion rootProject.ext.minSdkVersion
             targetSdkVersion rootProject.ext.targetSdkVersion
-            resValue "string", "app_name", "BetterSocial Dev"
+            resValue "string", "app_name", "HelioDev"
             applicationIdSuffix ".dev"
             resValue "string", "build_config_package", "org.bettersocial"
         }
         prod {
             minSdkVersion rootProject.ext.minSdkVersion
             targetSdkVersion rootProject.ext.targetSdkVersion
-            resValue "string", "app_name", "BetterSocial"
+            resValue "string", "app_name", "Helio"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The application name has been changed from "BetterSocial" to "Helio" for the prod build variant. This change is consistent with the PR title and should not cause any issues. However, please ensure that all references to the old app name are updated accordingly across the entire codebase to avoid any inconsistencies or potential crashes due to missing resources.


<!-- This is an auto-generated comment by OSS CodeRabbit -->